main.nf

```diff
@@ -612,10 +611,7 @@ if (verbose) bamsFMT2 = bamsFMT2.view {"Bams with Intervals for MuTect2: $it"}
 (bamsFFB, bamsAll, intervals) = generateIntervalsForVC(bamsAll, intervals)
 if (verbose) bamsFFB = bamsFFB.view {"Bams with Intervals for FreeBayes: $it"}
 
-// VarDict
-(bamsFVD, bamsAll, intervals) = generateIntervalsForVC(bamsAll, intervals)
-if (verbose) bamsFVD = bamsFVD.view {"Bams with Intervals for VarDict: $it"}
-
+// Manta and Strelka
```

Comment:
True, but I felt sad for Manta and Strelka, and I don't want them to be jealous of the other Variant Callers...
More seriously, I can totally remove that